### PR TITLE
Shared libs section in app list t145

### PR DESCRIPTION
**Pull request by OndraZizka**

### CODE REVIEW

File: graph/api/src/main/java/org/jboss/windup/graph/model/ProjectModel.java [hunk 1]
```diff
@@ -101,18 +106,32 @@
     void setVersion(String version);
 
     /**
-     * Contains the project's name.
+     * Project's name. This is most often derived from project's root filename.
```
Comment (jsight) [inline graph/api/src/main/java/org/jboss/windup/graph/model/ProjectModel.java]: I don't think it is accurate that this is generally derived from the root filename. Most are derived from the maven name or manifest.mf name.

Comment (OndraZizka) [inline graph/api/src/main/java/org/jboss/windup/graph/model/ProjectModel.java]: Fixed.
     * Project's name.
     * This is often derived Maven name or MANIFEST.MF name or from project's root filename.

**Comment by OndraZizka** [comment]
Rebased.

**Pull request merged**